lib/opensusebasetest.pm

```diff
@@ -482,6 +483,10 @@ sub enter_test_text {
     }
 }
 
+sub firewall {
+    return ((is_sle && !sle_version_at_least('15')) || (is_leap && !leap_version_at_least('15.0'))) ? 'SuSEfirewall2' : 'firewalld';
```

Comment:
Unfortunately PR was there for too long, so firewalld is now in TW as well. But as a good part, by virtue of good design, I guess it should be enough to adjust code here =)

Reply:
That's already incorporated. The if checks for *older* versions of SLE and Leap now and falls back to SuSEfirewall2, basically making firewallctl the new default now as I also recommended to @DimStar77

Reply:
You're right, sorry. Please, fix tidy and I'll merge: 
qw should be used as function at line 18, column 19.  use MODULE 'func' for single imports.  (Severity: 4)